tests/functional/generate_pages.py

```diff
@@ -854,6 +854,17 @@ def process_block(
                 page_filename=f'{repeating_block["id"]}-repeating-block.page.js',
             )
 
+    if block["type"] in {"ListCollector", "ListCollectorContent"}:
+        for repeating_block in block.get("repeating_blocks", []):
```

Comment:
This is duplicating the above, list collector is already handled. Handle repeating block for both, then handle add edit remove for list collector only